main/twai.h

```diff
@@ -16,7 +16,11 @@ static const twai_general_config_t g_config = {
     .intr_flags = ESP_INTR_FLAG_LEVEL1
 };
 static const twai_timing_config_t t_config = TWAI_TIMING_CONFIG_500KBITS();
-static const twai_filter_config_t f_config = TWAI_FILTER_CONFIG_ACCEPT_ALL();
+static const twai_filter_config_t f_config =  {
+    .acceptance_code = 0,
+    .acceptance_mask = 0xFFFFFFFF,
```

Comment:
@bri3d any recommendation on how this format works for like... "only give me arb IDs higher than 0x500" so we can filter out lots of low level traffic?

Reply:
https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/peripherals/twai.html#acceptance-filter

Note that the diagram is *backwards* - that is, the left side is bit 0 and the right side is bit 32. 

To say "only give me arb IDs higher than 0x500" I think we would use:

{.acceptance_code = (0x500 << 21),
.acceptance_mask = ~(0x500 << 21),

This would say "ensure that 8 and bit 10 in the Address are set, and _only_ bit 8 and bit 10."

To filter on _only_ ID 0x500, we'd instead use 0x5FF as the mask to say "all bits below 10 must match the code."

Reply:
This actually didn't work. Do you have anything else I can try?

Reply:
Now that I think about it, what I said didn't make any sense as an equality comparator. Do we want to match against only 0x7xx? That should be fairly easy to accomplish this way.